RELEASE.md

```diff
@@ -56,53 +56,69 @@ npm login
+  - In Manage mode on PyPI project, click `Publishing` and add new GitHub Trusted Publisher
+    - Set Workflow name to `publish-release.yml`
+    - Set Environment name to the environment your action will be using (Usually prod or test)
+  - Create access token for your User
+    - In Account Settings, click on `Add API token`
+    - Set its scope to your project only
+    - Save your PyPI token for when setting up your GitHub environment
+
+- Create GitHub Personal Access Token (PAT)
+
+  - Go to GitHub account `settings`
+  - Scroll down to `Developer Settings`
+  - Click `Personal Access Token`
+    - Select `Fine-grained tokens`
+  - `Generate new token` in the upper right
+  - Give the token a name and description
+  - Set `Resource Owner` to the GitHub organization or user who owns the repository
+  - Set `Repository Access` to Only select repositories
+    - Select your repository from the dropdown
+  - Give `Repository Permissions` read and write access to
+    - Contents
+    - Issues
+    - Pull Requests
+    - Workflows
+  - Save your token for when setting up your GitHub environment
+
+- Set up GitHub Environment
+  - Create a [GitHub Environment](https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-deployments/managing-environments-for-deployment)
+    - Go to your repository settings
+    - Under `Code and automation` select environment
+    - Click `New Environment` and provide a name
+  - In your GitHub Environment
+    - Add token to the [Github Secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) in the repository:
+      - `PUBLISH_GITHUB_PAT` (Your GitHub PAT)
+      - `PYPI_TOKEN` (Your PyPI token created in when setting up PyPI)
+    - Add to the [Github Variables](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables) in the repository:
+      - `REPOSITORY_URL` (The repository you will be uploading to)
+
+### Deploying Using GitHub Action
+
+- Go to GitHub Actions panel
+- Run with workflow_dispatch `Step 1: Prep Release`
+  - REQUIRED
+    - Set `environment` to the name the environment with your secrets and variables
+    - Set `version` to the new version of your package
+      - For alpha versions append "-alpha.X" to your version where X is your alpha iteration
+  - OPTIONAL
+    - Set `branch` to the branch you want to target
+    - Set `since` to the date or git reference you want to use the PRs since
+    - Set `since_last_stable` if you only want to use PRs since the last stable release
+    - Set `steps_to_skip` to a comma separated list of steps you want to skip in the release population step
+- Run with workflow_dispatch `Step 2: Publish Release`
+  - REQUIRED
+    - Set `environment` to the name the environment with your secrets and variables
+    - Set `tag` to the tag of the release you want to publish
+- If both jobs succeeded, your deployment to PyPI should be successful
 
```

Comment:
While not needed for this release, I think an example workflow based on the above instructions would be very helpful in seeing all the steps put together.